### PR TITLE
[Chore] remove old pay scale code that was supporting the scraper

### PR DESCRIPTION
**Pull request by despo**
## Trello card URL:
https://trello.com/c/ayPT9yYg/511-remove-old-pay-scale-code-that-was-supporting-the-scraper

## Changes in this PR:
- removes `expires_at`, `starts_at` and `salary` from `PayScale`
- removes `data:update:pay_scale` rake task
- removes update_pay_scale_task_command tf task
- updates `data:seed:pay_scale` to correctly set PayScale data without requiring update task

## Screenshots of changes:
screen_shot_2018-10-26_at_12_41_16
_*screenshot includes VacancyScraper related removed tasks_

## Next steps:
- [x] Terraform deployment required?

### CODE REVIEW

**Comment by despo** [comment]
[Remove old Pay Scale code that was supporting the scraper](https://trello.com/c/ayPT9yYg/511-remove-old-pay-scale-code-that-was-supporting-the-scraper)

File: lib/tasks/data/seed.rake [hunk 1]
```diff
@@ -17,55 +15,55 @@ namespace :data do
...
+PAYSCALE_DATA = [['MPS1', 'Main pay range 1'],
+                 ['MPS2', 'Main pay range 2'],
+                 ['MPS3', 'Main pay range 3'],
+                 ['MPS4', 'Main pay range 4'],
+                 ['MPS5', 'Main pay range 5'],
+                 ['MPS6', 'Main pay range 6'],
+                 ['UPS1', 'Upper pay range 1'],
+                 ['UPS2', 'Upper pay range 2'],
+                 ['UPS3', 'Upper pay range 3'],
+                 ['LPS1', 'Lead Practitioners range 1'],
```
Comment (tahb) [inline lib/tasks/data/seed.rake]: Should `Practitioners` also be down cased?

File: lib/tasks/data/seed.rake [hunk 2]
```diff
@@ -17,55 +15,55 @@ namespace :data do
...
-                        ['LPS35', 'Lead Practitioners Range 35', 90773],
-                        ['LPS36', 'Lead Practitioners Range 36', 93020],
-                        ['LPS37', 'Lead Practitioners Range 37', 95333],
-                        ['LPS38', 'Lead Practitioners Range 38', 97692],
-                        ['LPS39', 'Lead Practitioners Range 39', 100072],
-                        ['LPS40', 'Lead Practitioners Range 40', 102570],
-                        ['LPS41', 'Lead Practitioners Range 41', 105132],
-                        ['LPS42', 'Lead Practitioners Range 42', 107766],
-                        ['LPS43', 'Lead Practitioners Range 43', 109366]].freeze
+PAYSCALE_DATA = [['MPS1', 'Main pay range 1'],
```
Comment (tahb) [inline lib/tasks/data/seed.rake]: This used to be `Minimum Pay Range`, is this okay?

File: lib/tasks/data/seed.rake (hunk 1 above)
Comment (despo) [reply lib/tasks/data/seed.rake]: No, this is the data as it was provided by Michael and it matches staging (and thus production)

screen shot 2018-11-05 at 17 22 32 1

**Pull request merged**